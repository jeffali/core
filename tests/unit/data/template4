[%CFEngine BEGIN %]
First $(one) second$(foo.two) third$(invalid) fourth$(invalid:foo.one)
[%CFEngine END %]
