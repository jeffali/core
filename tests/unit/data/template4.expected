First one secondtwo third$(invalid) fourth$(invalid:foo.one)
